packages/codegen/src/configuration.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Cognite AS
+import { promises as fs } from 'fs';
+
+export type ConfigurationOptions = {
+  version: string;
+  service: string;
+  autoNameInlinedRequest: boolean;
+  scope: string;
```

Comment:
```suggestion
  scope: 'local' | 'global';
```

And maybe consider changing to `service` and `package` as scopes? I find `local` and `global` a bit confusing

Reply:
I like this idea